src/crypto/crypto_keys.cc

```diff
@@ -543,8 +543,7 @@ Maybe<bool> GetAsymmetricKeyDetail(
     case EVP_PKEY_EC: return GetEcKeyDetail(env, key, target);
     case EVP_PKEY_DH: return GetDhKeyDetail(env, key, target);
   }
-  THROW_ERR_CRYPTO_INVALID_KEYTYPE(env);
-  return Nothing<bool>();
+  return Just(false);
```

Comment:
What's the impact of this change on the other call site in `KeyObjectHandle::GetKeyDetail`? It checks `IsNothing` on the result of this function, which will never be true with this change.

Reply:
@tniessen would applying a change [like so](https://gist.github.com/panva/6e162cc98ede55a3e93097f08308d739) be acceptable? This would make this an entirely js side PR, no change to c++

Reply:
@panva That does seem like a good idea!

Reply:
will do